s4e-backend/src/main/java/pl/cyfronet/s4e/bean/Product.java

```diff
@@ -20,13 +23,16 @@
 public class Product {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @EqualsAndHashCode.Include
```

Comment:
It will be ignored when you don't add @EqualsAndHashCode(onlyExplicitlyIncluded = true) on the class level.